test/test-applications/integrations/TestApplication.Http.NetFramework/TestServer.cs

```diff
@@ -0,0 +1,136 @@
+// <copyright file="TestServer.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Net;
+using System.Text;
+using System.Threading;
+using TestApplication.Http.NetFramework.Helpers;
+
+namespace TestApplication.Http.NetFramework;
+
+public class TestServer : IDisposable
+{
+    private static readonly ActivitySource MyActivitySource = new ActivitySource("TestApplication.Http.NetFramework", "1.0.0");
+
+    private readonly HttpListener _listener;
+    private readonly Thread _listenerThread;
+    private readonly string _prefix;
+
+    public TestServer(string host, string sufix = "/")
```

Comment:
nit: refactor to:

```csharp
    public TestServer(string sufix)
```

host can be always `localhost` right?

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/1628/commits/e0b7ed1c3959d54de39f0a372259f3d04f4959f4